chroma-manager/tests/framework/utils/provisioner_interface/release_cluster

```diff
@@ -3,10 +3,11 @@
 PROVISIONER=${PROVISIONER:?"Need to specify a PROVISIONER string"}
 
 # cluster_provisioned protects us from releasing the same cluster twice
-if [ -f cluster_provisioned ]; then
+echo "$PWD" >&2
+if [ -f "$WORKSPACE"/cluster_provisioned ]; then
     if ! sed -e 's/provision\": *true/provision\":false/g' < provisioner_output.json | $PROVISIONER; then
-        rc=$?
-        echo "unprovisioning failed" >&2
+        rc="${PIPESTATUS[0]}"
```

Comment:
where does PIPESTATUS get populated ?

Reply:
http://tldp.org/LDP/abs/html/internalvariables.html and then search for PIPESTATUS for a detailed explanation, but it's a variable that is populated after every foreground pipeline with the exit status of each element of the pipeline.  It's a better `$?` since it reflects each command in the pipeline.  It's better than `$?` because it is immune from having it's meaning changed when the pipeline is extended and/or when you don't want the exit code of just the last item in the pipeline.